hs_core/models.py

```diff
@@ -2240,6 +2276,9 @@ def __check_irods_directory(self, dir, logger,
                     ecount += 1
                     msg = "check_irods_files: file {} in iRODs does not exist in Django"\
                         .format(fullpath)
+                    if clean:
+                        istorage.delete(fullpath)
```

Comment:
@alvacouch Probably a try...except block should be added around ```istorage.delete(fullpath)``` to catch iRODS exceptions in case delete fails on iRODS side before printed success message subsequently?